homeassistant/components/utility_meter/__init__.py

```diff
@@ -192,7 +188,7 @@ async def async_reset_meters(service_call):
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Utility Meter from a config entry."""
 
-    await async_remove_stale_device_links(
+    d.async_remove_stale_devices_links_keep_entity_device(
```

Comment:
Let's import this method directly, we don't have to import devices as an alias in this case.